net/calendar-contacts-server/Portfile

```diff
@@ -1,71 +1,73 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem              1.0
-PortGroup               github 1.0
-PortGroup               active_variants 1.1
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           active_variants 1.1
 
-github.setup            apple ccs-calendarserver 50894b7
+github.setup        apple ccs-calendarserver 13c706b985fb728b9aab42dc0fef85aae21921c3
 
-name                    calendar-contacts-server
+name                calendar-contacts-server
 # version from https://github.com/apple/ccs-calendarserver/blob/master/setup.py
 # with date of git commit appended
-version                 9.3.20190916
-revision                0
-categories              net mail
-platforms               darwin
-supported_archs         noarch
-maintainers             {ieee.org:s.t.smith @essandess} openmaintainer
-license                 Apache-2
-
-description             Apple Calendar and Contacts Server configuration
-long_description ${description}. \
-    ccs-calendarserver is a standards-compliant server implementing\
-    the CalDAV and CardDAV protocols, including iMIP and APNS. It\
-    provides a shared location on the network allowing multiple users\
-    to store and edit calendaring and contact information. This port\
-    provides a basic, working, easily modifiable configuration,\
-    previously used in macOS Server.app, and an nginx reverse proxy\
-    to handle modern crypto and isolate the backend server.
-
-homepage                https://www.calendarserver.org
-
-checksums               rmd160  fd77e68c4cf6dceb543ce8cff47c5f6c23429ed6 \
-                        sha256  c97384bb26cec6a764eba3c8cd590c63e912c93f85e8fd5ce3945e6e11065dae \
-                        size    3714315
+version             9.3.20200212
+revision            0
+categories          net mail
+platforms           darwin
+supported_archs     noarch
+maintainers         {ieee.org:s.t.smith @essandess} openmaintainer
+license             Apache-2
+
+description         Apple Calendar and Contacts Server configuration
+long_description    ccs-calendarserver is a standards-compliant \
+                        server implementing the CalDAV and CardDAV \
+                        protocols, including iMIP and APNS. It \
+                        provides a shared location on the network \
+                        allowing multiple users to store and edit \
+                        calendaring and contact information. This port \
+                        provides a basic, working, easily modifiable \
+                        configuration, previously used in macOS \
+                        Server.app, and an nginx reverse proxy to \
+                        handle modern crypto and isolate the backend \
+                        server.
```

Comment:
please align this the same as the first line on the `long_description` and make white-space changes in a separate commit not mixed in with functional changes

Reply:
Thanks done.

Reply:
there are still a lot of other white-space changes and/or reordering of lines happening in the other commit

Reply:
Yes, I see I messed that up.

Reply:
Fixed now.